assets/css/style.css

```diff
@@ -205,6 +207,36 @@ header {
     transition: max-height 0.2s ease-out;
 }
 
+.main3{
+    margin-top:50px;
+    color:#3f3844;
+}.main3 .image-color p{
+    color:white;
+    font-size: 26px;
+    text-align: center;
+}
+
+.text{
+    text-align: center;
+} h2{
+    margin-bottom: 20px;
+    font-size: 35px;
+}.text p{
+    font-size: 18px;
+}
+
+.image-color{
+    background: linear-gradient(#4ECDC4, #1A535C);
+    margin-left: 30px;
+    border-radius: 10px;
+    margin-top: 30px;
+}.image-color img{
+    margin-left: 20px;
+    position: relative;
```

Comment:
you can use `left` and `transform` instead of using `margin-left`.

```
.image-color img {
    position: relative;
    left: 50%;
    transform: translateX(-50%);
}
```